enzy_htp/structure/structure_io/pdb_io.py

```diff
@@ -0,0 +1,554 @@
+"""Generation/construction of Structure objects from PDB files and exporting it vice versa
+Definition of PDB file format (http://www.wwpdb.org/documentation/file-format-content/format33/v3.3.html)
+
+Author: shaoqz, <shaoqz@icloud.com>
+Date: 2022-08-01
+"""
+from collections import defaultdict
+import os
+import string
+import sys
+from typing import Dict, List, Tuple, Union
+from plum import dispatch
+from biopandas.pdb import PandasPdb
+import pandas as pd
+
+
+from ._interface import StructureParserInterface
+from ..atom import Atom
+from ..metal_atom import MetalUnit, residue_to_metal
+from ..residue import Residue
+from ..solvent import Solvent, residue_to_solvent
+from ..ligand import Ligand, residue_to_ligand
+from ..chain import Chain
+from ..structure import Structure
+from enzy_htp.core import _LOGGER
+import enzy_htp.core.file_system as fs
+import enzy_htp.chemical as chem
+
+# pylint: disable=logging-fstring-interpolation
+class PDBParser(StructureParserInterface):
+    '''
+    Parser for covert PDB into Structure and vice versa
+    '''
+    def __init__(self) -> None: # pylint: disable=super-init-not-called
+        '''
+        not sure what can be init here. Maybe some configuration?
```

Comment:
In terms of what to initialize here, I think a good model is to make it have all arguments about parsing configuration that default to pre-determined values

Reply:
got it but the thing is there is no configuration right now just a place holder here.

Reply:
Got it! There's nothing wrong with it, just sharing my thoughts :)

Reply:
Cool! Got it!